cygnus-ngsi-ld/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIPostgisSink.java

```diff
@@ -268,9 +269,7 @@ public void stop() {
     @Override
     public void start() {
         try {
-            if (buildDBName(null) != null) {
-                createPersistenceBackend(postgisHost, postgisPort, postgisUsername, postgisPassword, maxPoolSize, buildDBName(null), postgisOptions);
-            }
+            createPersistenceBackend(postgisHost, postgisPort, postgisUsername, postgisPassword, maxPoolSize, postgisOptions);
```

Comment:
Note that ngsi-ld sinks like this are not fully aligned with ngsiv2 sinks and maybe are not received latest features, bugfixes and refactors.